release-tools/src/main/java/org/springframework/data/release/build/MavenBuildSystem.java

```diff
@@ -341,9 +389,10 @@ public void verify() {
 		CommandLine arguments = CommandLine.of(Goal.CLEAN, Goal.VERIFY, //
 				profile("central"), //
 				SKIP_TESTS, //
+				arg("maven.repo.local").withValue("~/temp/spring-data-shell/repository"), //
```

Comment:
I wonder why this is necessary as we default for the other commands to `~/temp/spring-data-shell/repository` as local repo. See `MavenRuntime` at `setLocalRepositoryDirectory(…)`.

Reply:
I think this was a relic from debugging. Removed.